sound/soc/intel/boards/sof_pcm512x.c

```diff
@@ -45,6 +45,8 @@ struct sof_card_private {
 	struct list_head hdmi_pcm_list;
 	bool idisp_codec;
 	struct gpio_desc *gpio_4;
+	struct clk *sclk;
+	bool is_dac_pro;
```

Comment:
We have multiple HiFiBerry models, right? Maybe an `enum`?

Reply:
no, we only have two